config/manager/manager.yaml

```diff
@@ -27,7 +27,7 @@ spec:
         - /manager
         args:
         - --enable-leader-election
-        image: quay.io/opentelemetry/opentelemetry-operator:master
+        image: controller
```

Comment:
Why has this changed back from referencing a real image? Is it only used locally?